explore/index.html

```diff
@@ -21,22 +21,112 @@ <h4 class="subheader">OME products are in use in labs and
         <!-- end intro section -->
         
         <!-- begin section for OMERO for Scientists -->
-        <!-- Liza to add OMERO, Imagej2, IDR and an OMERO.figure logo/same graphics as used to the figure feature, and reformat as needed -->
         <hr class="whitespace">
         <div class="row column text-center">
             <div class="small-12 columns">
                 <h2>OME for Scientists</h2>
-                <h5 class="subheader">Try out OME products with your own
-                    microscopy data
-                    </h5>
+                <h5 class="subheader">Read any image data from Java, <a href="https://imagej.net/Welcome" target="_blank">Image</a>/<a href="https://fiji.sc/" target="_blank">Fiji</a>, GNU Octave/MATLAB, Python, R with Bio-Formats.</h5>
             </div>
         </div>
         <hr>
         <div class="row column text-center">
-            <p>Sign up for our <a href="http://help.openmicroscopy.org/demo-server.html">OMERO demo server</a> to access the OMERO clients and test functionality like OMERO.figure.</p>
+            <p>Sign up for our <a href="http://help.openmicroscopy.org/demo-server.html">demo server</a> to test out the functionality with your own data.</p>
+            <p>Already using ImageJ/Fiji? Check out the <a href="https://www.openmicroscopy.org/site/support/bio-formats5.4/users/index.html#using-bio-formats-with-imagej-and-fiji">Bio-Formats</a> plugin.</p>
```

Comment:
Can we update this to:
"Bio-Formats [product page link] lets you read image data from Java, ImageJ/Fiji [with the links], GNU Octave/MATLAB, Python and R. If you're already using Fiji, it comes bundled with the Bio-Formats [https://www.openmicroscopy.org/site/support/bio-formats/users/index.html#using-bio-formats-with-imagej-and-fiji] plugin automatically."